src/components/AudioCell.vue

```diff
@@ -121,6 +131,80 @@ const audioItem = computed(() => store.state.audioItems[props.audioKey]);
 
 const uiLocked = computed(() => store.getters.UI_LOCKED);
 
+const isMultiSelectEnabled = computed(
+  () => store.state.experimentalSetting.enableMultiSelect
+);
+
+const setActiveAudioKey = () => {
+  if (store.getters.ACTIVE_AUDIO_KEY !== props.audioKey) {
+    store.dispatch("SET_ACTIVE_AUDIO_KEY", { audioKey: props.audioKey });
+    store.dispatch("SET_SELECTED_AUDIO_KEYS", { audioKeys: [props.audioKey] });
+  }
+};
+const onClickWithModifierKey = (event: MouseEvent) => {
```

Comment:
これも関数名からいつ呼ばれるかはわかるけれども、何をするためのものなのか一瞬わからないのでコメントしておくのどうでしょう。
```suggestion
// 複数選択クリック
const onClickWithModifierKey = (event: MouseEvent) => {
```

Reply:
コメントを足しました。